scheduler/data-flow/src/main/kotlin/io/seldon/dataflow/kafka/PipelineStatus.kt

```diff
@@ -0,0 +1,93 @@
+package io.seldon.dataflow.kafka
+
+import io.klogging.Klogger
+import io.klogging.Level
+import io.klogging.NoCoLogger
+import io.seldon.dataflow.DataflowStatus
+import kotlinx.coroutines.runBlocking
+import org.apache.kafka.streams.KafkaStreams
+
+open class PipelineStatus(val state: KafkaStreams.State?, var isError: Boolean) : DataflowStatus {
```

Comment:
question: where `state` be set outside of errors?

Reply:
For example `StreamStarted` and `StreamStopped` are not errors by default (the Pipeline transitions to `StreamStopped` after the stop() call, which would also happen when the pipeline gets deleted, for example).